datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -7106,6 +7131,31 @@ input DescriptionUpdateInput {
     subResource: String
 }
 
+"""
+Updates the label of a resource. Currently supports DatasetField labels only
+"""
```

Comment:
Label is maybe a bit too generic of a name here. This is strictly restricted to Dataset Fields, and should be named as such. 

Consider DatasetFieldLabelUpdateInput

Reply:
Sounds good

Reply:
Addressed